src/lib/field/Editor/multi.vue

```diff
@@ -3,7 +3,7 @@
     <div v-if="singleInput" class="mb-2">
       <slot name="single"></slot>
     </div>
-    <draggable :list.sync="val">
+    <draggable :list.sync="val" handle=".handle">
```

Comment:
`handle=".handle"` -- is this a typo?

Reply:
No, makes it so you can drag and drop by the element with the class .handle instead of being able to drag/drop everywhere in the draggable list.